src/modules/roc_netio/target_libuv/roc_netio/udp_sender_port.cpp

```diff
@@ -142,8 +158,24 @@ void UdpSenderPort::write(const packet::PacketPtr& pp) {
             return;
         }
 
+        if (rate_limiter_.allow()) {
+            const double nb_ratio = nb_packet_counter_ != 0
+                ? (double)nb_packet_counter_ / packet_counter_
+                : 0.;
```

Comment:
If we want to avoid division by zero, we should check `packet_counter_ != 0` instead of `nb_packet_counter_ != 0`.